dias/utils/tracker.py

```diff
@@ -262,14 +262,17 @@ def insert_files(self, files):
 
         file_type_rows = self._get_filetypes()
 
-        self.logger.info("Adding {0} to database".format(len(files)))
+        self.logger.debug("Adding {0} DIAS input files to database".format(len(files)))
 
         # remove already logged files from list
         files_in_table = [f.filepath for f in File.select()]
 
         files = [f for f in files if f not in files_in_table]
 
         for f in files:
+            if Path(f + ".lock").is_file():
```

Comment:
```suggestion
            if Path("." + f + ".lock").is_file():
```

The lock file pattern is `.<filename>.lock` (with a leading dot, to hide it from `ls`).

Reply:
Oh, but my suggestion isn't going to work if there are path elements in `f`.

Reply:
> Oh, but my suggestion isn't going to work if there are path elements in `f`.

You can use `Path.name` and `Path.parent`

Reply:
Nice catch, Don!